src/applications/gi/utils/render.jsx

```diff
@@ -3,25 +3,46 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
 export const renderSchoolClosingAlert = result => {
   const { schoolClosing } = result;
+
   if (!schoolClosing) return null;
   return (
     <AlertBox
-      content={<p>Upcoming campus closure</p>}
-      headline="School closure"
+      content={<li>Upcoming campus closure</li>}
+      headline="A campus is closing soon"
       isVisible={!!schoolClosing}
       status="warning"
     />
   );
 };
 
+const renderReasons = cautionFlags => {
+  const flags = [];
+  cautionFlags
+    .sort((a, b) => {
+      if (a.reason.toLowerCase() < b.reason.toLowerCase()) return -1;
+      if (a.reason.toLowerCase() > b.reason.toLowerCase()) return 1;
+      return 0;
+    })
+    .forEach(flag => {
+      flags.push(<li key={flag.id}>{flag.reason}</li>);
+    });
+
+  return flags;
+};
+
 export const renderCautionAlert = result => {
-  const { cautionFlag } = result;
-  if (!cautionFlag) return null;
+  const { cautionFlags } = result;
+
+  if (cautionFlags.length > 0) return null;
```

Comment:
Won't this return null, if there are caution flags?

Reply:
After talking with Matt, pushed a fix for this here.